src/clientRoutes.js

```diff
@@ -342,6 +342,14 @@ const handleV2Sweep = co(function *handleV2Sweep(req) {
   return wallet.sweep(req.body);
 });
 
+// handle CPFP accelerate transaction creation
+const handleV2AccelerateTransaction = co(function *handleV2BumpTransaction(req) {
```

Comment:
Is the "handleV2BumpTransaction" still accurate? Shouldn't it be "…accelerateTx"?

Reply:
Done. See #170